onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -129,19 +129,22 @@ struct OrtStatus {
 #include "core/providers/cpu/cpu_execution_provider.h"
 #include "core/providers/cpu/cpu_provider_factory.h"
 
+#if defined(USE_CUDA) || defined(USE_ROCM)
 #ifdef USE_CUDA
 #include "core/providers/cuda/cuda_provider_factory.h"
 #include "core/providers/cuda/shared_inc/cuda_call.h"
 #include "core/providers/cuda/cuda_execution_provider.h"
 #include "core/providers/cuda/cuda_allocator.h"
-
-OrtDevice::DeviceId cuda_device_id = 0;
 OrtCudnnConvAlgoSearch cudnn_conv_algo_search = OrtCudnnConvAlgoSearch::EXHAUSTIVE;
+bool do_copy_in_default_stream = true;
```

Comment:
Can we simply implement this for ROCm as well? (Seems like the same bug that this is supposed to address might bite us on AMD as well....)

Reply:
Yes, I was thinking to do it in a different PR.